enclave-runner/src/library.rs

```diff
@@ -8,19 +8,45 @@ use std::path::Path;
 use std::sync::Arc;
 
 use failure::Error;
-use sgxs::loader::Load;
+use sgxs::loader::{Load, MappingInfo};
 
 use loader::{EnclaveBuilder, ErasedTcs};
+use std::fmt;
+use std::os::raw::c_void;
 use usercalls::EnclaveState;
 
 pub struct Library {
     enclave: Arc<EnclaveState>,
+    address: usize,
+    size: usize,
+}
+
+impl fmt::Debug for Library {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Use the [debug struct formatter](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct)